rs/sns/testing/scripts/env.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+
+bazel_targets=(
+    "//rs/sns/testing:sns-testing-init"
+    "//rs/sns/testing:sns-testing"
+    # TODO: remove targets below once the suggested way to get them is changed
+    # to downloading them from CDN
+    "//rs/sns/cli:sns"
+    "//rs/pocket_ic_server:pocket-ic-server"
+)
+
+bazel_root="$(bazel info workspace)"
+bin_dir="$PWD/bin"
```

Comment:
bin_dir should probably be an optional argument here, so that you can install them in a semi-stable location (and not necessarily related to where you happen to run the command from)

Reply:
I think eventually these binaries will be distributed without `bazel`, which will make this script obsolete. But for now, this script exists for the sake of README simplification by avoiding prefixing each command with `bazel run`, and since currently the `sns-testing` README requires running commands from the `sns-testing` crate root, the existing approach should be good enough